src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_validators.py

```diff
@@ -116,8 +116,9 @@ def datetime_type(string):
     return datetime.strptime(string, date_format)
 
 def vault_base_url_type(name):
-    from azure.cli.core._profile import Profile
-    from azure.cli.core._azure_env import ENDPOINT_URLS
-    profile = Profile()
-    suffix = profile.env[ENDPOINT_URLS.KEY_VAULT].replace('https://', '.')
-    return 'https://{}{}'.format(name, suffix)
+    #from azure.cli.core._profile import Profile
+    #from azure.cli.core._azure_env import ENDPOINT_URLS
+    #profile = Profile()
+    #suffix = profile.env[ENDPOINT_URLS.KEY_VAULT].replace('https://', '.')
+    #return 'https://{}{}'.format(name, suffix)
+    return 'https://{}.vault.azure.net'.format(name)
```

Comment:
Why is this now less generic than what we had before (thinking of multi-cloud scenario)?
Also, remove the commented code above?